prow/cluster/jobs/istio/test-infra/istio.test-infra.master.yaml

```diff
@@ -58,6 +59,24 @@ presubmits:
       # hostPath:
       #   path: /mnt/disks/ssd0/test-infra
 
+  - name: pull-test-infra-prow-checkconfig
```

Comment:
Do we need to do this in a new job? Can we instead make it part of the bazel presubmit?

Reply:
I think we'd need to vendor the `checkconfig` utility to do that? This uses the `checkconfig` image instead so that we can easily keep the `checkconfig` version in sync with the Prow components.